frontend/src/concepts/pipelines/content/createRun/RunForm.tsx

```diff
@@ -75,6 +107,9 @@ const RunForm: React.FC<RunFormProps> = ({ data, onValueChange, isCloned }) => {
           data={data.nameDesc}
           setData={(nameDesc) => onValueChange('nameDesc', nameDesc)}
           maxLength={RUN_NAME_CHARACTER_LIMIT}
+          onNameBlur={onNameBlur}
```

Comment:
can you use lodash debounce instead. a user wouldn't see the helper text unless if they never click out

Reply:
or scroll down and click out

Reply:
A debounce of how many seconds would you like?

Reply:
the `useDebounceCallback` hook uses 250ms